code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs

```diff
@@ -297,9 +301,8 @@ userCharacteristicsIntro = foldlSP
 
 probDescIntro :: Sentence
 probDescIntro = foldlSent_
-  [S "create a", foldlList Comma List $ map S ["simple", "lightweight", "fast", "portable"],
-  getAcc twoD, phrase CP.rigidBody, phrase physLib `sC` S "which will allow for more accessible",
-  phrase game, S "development" `S.and_` S "the production of higher quality" +:+. plural product_,
+  [purp `sC` S "in a", foldlList Comma List $ map S ["simple", "lightweight", "fast", "portable"],
```

Comment:
I'm pretty sure there shouldn't be a comma here
```suggestion
  [purp, S "in a", foldlList Comma List $ map S ["simple", "lightweight", "fast", "portable"],
```

Reply:
That does read better, thanks!

Reply:
(Just make sure that stable also gets updated 😅)

Reply:
Right, I also forget about stable when accepting suggested changes on GitHub... 😅